.github/scripts/print_pr_details.py

```diff
@@ -0,0 +1,55 @@
+# .github/scripts/print_pr_details.py
+from github import Github
+import os
+
+# Initialize the GitHub client
+token = os.getenv('GITHUB_TOKEN')
+g = Github(token)
+
+# Get repository information from environment variables
+repo_name = os.getenv('GITHUB_REPOSITORY')
+pr_number = os.getenv('GITHUB_REF').split('/')[-2]  # Extract PR number from GitHub ref
```

Comment:
The PR number extraction from GITHUB_REF can fail if the ref format changes. Consider adding error handling for this case.